OpenRA.Mods.Common/Traits/EntersTunnels.cs

```diff
@@ -0,0 +1,133 @@
+#region Copyright & License Information
+/*
+ * Copyright 2007-2017 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+using System.Collections.Generic;
+using System.Drawing;
+using System.Linq;
+using OpenRA.Mods.Common.Activities;
+using OpenRA.Mods.Common.Orders;
+using OpenRA.Traits;
+
+namespace OpenRA.Mods.Common.Traits
+{
+	[Desc("Enables the enter cursor to issue a move order through a tunnel.")]
```

Comment:
This wording is awkward and implies that cursors are in charge of issuing orders (which I don't think is the case, but could be wrong?).

How about `This actor can be ordered to move through a tunnel.`?